server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/TemplateGroupsAction.java

```diff
@@ -83,7 +85,10 @@ public void define(WebService.NewController context) {
         "When this parameter is not set, only group having at least one permission are returned.")
       .setExampleValue("eri");
 
-    createProjectPermissionParameter(action);
```

Comment:
suggestion: keep the method and add isRequired(false)

Reply:
done, method restored w a parameter for mandatory